fastly/block_fastly_service_v1_settings.go

```diff
@@ -25,6 +25,17 @@ func (h *SettingsServiceAttributeHandler) Process(d *schema.ResourceData, latest
 		DefaultTTL: uint(d.Get("default_ttl").(int)),
```

Comment:
```suggestion
		DefaultTTL:      uint(d.Get("default_ttl").(int)),
		StaleIfErrorTTL: gofastly.Uint(uint(d.Get("stale_if_error_ttl").(int))),
```

Reply:
Can you also update the comment accordingly?
https://github.com/fastly/terraform-provider-fastly/blob/v0.35.0/fastly/block_fastly_service_v1_settings.go#L24-L25
something like:

```
		// We have the same default value that is provided by
		// the Fastly API, so it's safe to include here
```